docs/architecture/grpc-for-wcf-developers/docker.md

```diff
@@ -24,10 +24,9 @@ For each image, there are four variants based on different Linux distributions,
 
 | Image tag(s) | Linux | Notes |
 | --------- | ----- | ----- |
-| 5.0-buster, 5.0 | Debian 10 | The default image if no OS variant is specified. |
-| 5.0-alpine | Alpine 3.9 | Alpine base images are much smaller than Debian or Ubuntu ones. |
-| 5.0-disco | Ubuntu 19.04 | |
-| 5.0-bionic | Ubuntu 18.04 | |
+| 5.0-buster-slim, 5.0 | Debian 10 | The default image if no OS variant is specified. |
+| 5.0-alpine | Alpine 3.12 | Alpine base images are much smaller than Debian or Ubuntu ones. |
+| 5.0-focal| Ubuntu 20.04 | |
```

Comment:
Consider using the Docs Authoring Pack - Visual Studio Code extension if you're not already. There are plenty of really good features that add support for common MD scenarios, one of them specifically is the formatting or consolidation of tables for readability. For more information, see [Reformat Markdown tables](https://docs.microsoft.com/en-us/contribute/docs-authoring/reformat-table).

Using the **Evenly distribute selected table** open when you select the table:

```md
| Image tag(s)         | Linux        | Notes                                                           |
|----------------------|--------------|-----------------------------------------------------------------|
| 5.0-buster-slim, 5.0 | Debian 10    | The default image if no OS variant is specified.                |
| 5.0-alpine           | Alpine 3.12  | Alpine base images are much smaller than Debian or Ubuntu ones. |
| 5.0-focal            | Ubuntu 20.04 |                                                                 |
```

This formatting makes the table easier to maintain, and read in its MD form.

Reply:
This extension looks to be cool. No, I wasn't using that but I will use now. Thank you for sharing 👍